src/lib/libpthread.js

```diff
@@ -979,11 +979,6 @@ var LibraryPThread = {
   $establishStackSpace__internal: true,
   $establishStackSpace__deps: ['$stackRestore', 'emscripten_stack_set_limits'],
   $establishStackSpace: (pthread_ptr) => {
-#if ALLOW_MEMORY_GROWTH
-    // If memory growth is enabled, the memory views may have gotten out of date,
-    // so resync them before accessing the pthread ptr below.
-    updateMemoryViews();
-#endif
```

Comment:
Was this a special case that was never needed?

Reply:
Yeah: https://github.com/emscripten-core/emscripten/pull/24295/commits/333c9f4cc3cb1e8c0f48408e0ee8da41d386f154